datastructure.c

```diff
@@ -112,7 +136,8 @@ int findForwardID(const char * forward, bool count)
 	else
 		forwarded[forwardID].count = 0;
 	// Save forward destination IP address
-	forwarded[forwardID].ip = strdup(forward);
+	// Store client IP
```

Comment:
Is this saving the client IP? Maybe delete this comment and use the above comment.